prboom2/src/m_menu.c

```diff
@@ -5214,6 +5214,12 @@ dboolean M_Responder (event_t* ev) {
       ch = 3;
     else if (ev->data1 & 16)
       ch = 4;
+    else if (ev->data1 & 32)
```

Comment:
How about this?
https://github.com/fabiangreffrath/woof/blob/f0c5d0733f005b97cf7e69055573ef49914df7e4/Source/m_menu.c#L5034
https://github.com/fabiangreffrath/woof/blob/f0c5d0733f005b97cf7e69055573ef49914df7e4/Source/m_menu.c#L4354

Reply:
done